packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -699,12 +699,11 @@ function wait(time) {
  */
 async function getAllTaxonomies(payload, skip = 0, taxonomies = []) {
   payload['type'] = 'taxonomies';
-  const response = await taxonomySDKHandler(payload, skip);
-  if (response) {
+  const { items, count } = await taxonomySDKHandler(payload, skip);
+  if (items) {
     skip += payload.limit;
-    //TODO - replace response.taxonomies with items
-    taxonomies = [...taxonomies, ...response.taxonomies];
-    if (skip >= response?.count) {
+    taxonomies = [...taxonomies, ...items];
```

Comment:
Is it necessary to spread both arrays.? The spread operator is equivalent to `for..of` loop [`O(n)`]. Instead, can we do `taxonomies.push(...items)`

Reply:
@antonyagustine sure, in this case push is better than spread operator. Done